source/change-streams/tests/change-streams.yml

```diff
@@ -0,0 +1,391 @@
+collection_name: &collection_name "test"
+database_name: &database_name "change-stream-tests"
+collection_name_2: &collection_name_2 "test2"
+database_name_2: &database_name_2 "change-stream-tests-2"
+tests:
+  -
+    description: The watch helper must not throw a custom exception when executed against a single server topology, but instead depend on a server error
+    minServerVersion: "3.6.0"
+    target: collection
+    topology:
+      - single
```

Comment:
I assume this would also fail if executed against a `mongos` cluster where individual shards are not replica sets. FWIW, I believe this is currently supported but in the future sharded clusters may require replica sets, so we may not care about it.